types/address.go

```diff
@@ -687,6 +687,7 @@ func GetPubKeyFromBech32(pkt Bech32PubKeyType, pubkeyStr string) (crypto.PubKey,
 
 	// We are bech32ifying some secp256k1 keys in tests.
 	case *secp256k1.PubKey:
+		protoPk = aminoPk
```

Comment:
This was causing a 038->039 migrate test failure. When running a migration from v038 to v039 state, we need to unmarshal a bech32 pubkey string into a secp256k1 PubKey.